src/basics.md

```diff
@@ -1188,6 +1189,48 @@ Error level - description
 
 Run the recipe with `RUST_BACKTRACE=1` to display a detailed [`backtrace`] associated with this error.
 
+[ex-measure-elapsed-time]: #ex-measure-elapsed-time
+<a name="ex-measure-elapsed-time"></a>
+## Measure the elapsed time between two code sections
+
+[![std-badge]][std] [![cat-time-badge]][cat-time]
+
+One way to get the elapsed time is to directly use the method 
```

Comment:
Let's reformat the description somewhat to a form like "Measures [`time::Instant::elapsed`] since [`time::Instant::now`]"